sphinxext/rediraffe.py

```diff
@@ -124,10 +125,11 @@ def build_redirects(app: Sphinx, exception: Union[Exception, None]) -> None:
     """
     Build amd write redirects
     """
-    try:
-        app.env.redirected
-    except AttributeError:
-        app.env.redirected = {}
+    redirect_json_file = Path(app.outdir) / "redirect.json"
```

Comment:
Overall, looks good to me.

The one thing that sticks out is `"redirect.json"`. Would something like `"rediraffe_record.json"`or `"rediraffe_cache.json"` be a better name? @Daltz333 what do you think?

Reply:
Good point, I've now changed the name